### PR TITLE
Bugfix/click outside errors

### PR DESCRIPTION
**Pull request by CoffeeGinger**
# Submit a pull request

Please make sure the following is true:

- [x] This is not a duplicate of another PR
- [x] The correct target branch selected?
- Breaking changes
  - [x] No existing features have been broken (without good reason)
  - [ ] PR introduces breaking changes
- [x] Commit messages are meaningful
- [x] The behaviour is as the documentation describes, or you updated the docs
- Tests
  - [x] Tests have been added/updated for new/modified unit-testable functions/helpers
  - [x] Test were run and did pass
- [x] The linter was run and did pass

## PR details

Remove unused onClickOutside HOC from currencycell as it threw lots of errors on scroll.

## Breaking changes

- What is broken, and why

## Other information/comments

- Anything to tell the world? :D

### CODE REVIEW

File: src/app/components/cells/currency/CurrencyCell.jsx [hunk 1]
```diff
@@ -72,6 +71,8 @@ class CurrencyCell extends React.PureComponent {
     this.exitCurrencyCell();
   };
 
+  onClickOutside = () => {};
```
Comment (hermann-p) [inline src/app/components/cells/currency/CurrencyCell.jsx:74]: Warum muss das hier rein, wenn der clickOutside ja rauskommt?

File: src/app/redux/actionCreators.js [hunk 2]
```diff
@@ -659,11 +658,6 @@ const actionCreators = {
   addSkeletonColumns: dispatchParamsFor(COLUMNS_DATA_LOADED),
   addSkeletonRow: dispatchParamsFor(ADDITIONAL_ROWS_DATA_LOADED),
   toggleCellSelection: data => {
-    const {
-      selectedCell: {
-        selectedCell: { langtag }
-      }
-    } = store.getState();
```
Comment (hermann-p) [inline src/app/redux/actionCreators.js:666]: War das noch übrig vom schnelleren Zellklick?

**Comment by hermann-p** [comment]
Reminder: Here be comments

File: src/app/components/cells/currency/CurrencyCell.jsx (hunk 1 above)
Comment (CoffeeGinger) [reply src/app/components/cells/currency/CurrencyCell.jsx]: Muss es nicht, hatte ich nur übersehen

**Review by hermann-p — APPROVED** [review]

**Pull request merged**